docs/support/tutorials/python-usage-guide.md

```diff
@@ -0,0 +1,254 @@
+# Using Python on CSC supercomputers
+
+Some important aspects of working with the Python programming language
+are notably different on CSC supercomputers compared to usage on a personal
+device or in other HPC environments. To make the most of the computational
+resources available to you, it is helpful to be aware of the differences.
+
+See the
+[Python application page](../../apps/python.md)
+for general information on the Python language and
+pre-installed Python environments on CSC supercomputers.
+
+## Creating and managing Python environments
+
+### Installing Python packages to existing modules
+
+If there is a CSC-provided module that covers _almost_ everything you
+need, but it is missing a few Python packages, you can try installing
+those yourself with the pip package manager.
+
+See the [package lists on our Python application
+page](../../apps/python.md#pre-installed-python-environments) to find out
+which packages are installed in existing modules.
+If you think that some important package should be included by default
+in a module provided by CSC, don't hesitate to contact our [Service
+Desk](../contact.md).
+
+=== "Using `venv`"
```

Comment:
As a usability/cosmetic issue I find it confusing with the tab as you can't see what part of the text is the one that will change. Maybe there could be some line or border around the area that belongs to the "tabbable" area?

Reply:
Good point, I was wondering about that too. All tabbed content is now inside "default" type admonitions.

Reply:
Now it much clearer what belongs to the tab. Those boxes look a bit "heavy", but I don't know of a better solution. Maybe ask @joonas-somero ?

Reply:
How about a simple horizontal rule? Like this:

```markdown
---

## Creating your own Python environments
```

Reply:
Also, should the admonitions have titles? Instead of just `!!! info`, perhaps for example `!!! info "Module compatibility"` would be more appropriate.

Reply:
> How about a simple horizontal rule? Like this:

This is what it looks like:
![image](https://github.com/CSCfi/csc-user-guide/assets/50655931/86a0d9a6-83af-4e2b-9049-defb64e593bc)

Reply:
@rkronberg suggested removing the nested admonitions, which I did. The boxes look much more visually pleasing to me now. Do we feel there still a need for using something else (e.g. the horizontal rule @joonas-somero suggested)?

![image](https://github.com/CSCfi/csc-user-guide/assets/91546703/7a7f7d0d-c78c-4def-879c-4ff63e3c586e)

Reply:
The admonitions are in danger of becoming severely overused and using them merely as a border looks, IMO, really heavy. How about a CSS solution? Here's one with a light border:
![image](https://github.com/CSCfi/csc-user-guide/assets/50655931/44fc660a-6170-4a78-9998-fd3edfa5ec9c)

Reply:
Okay, makes sense. I replaced the admonitions with a horizontal rule, since it matches nicely with the automatically created rule at the start of the tab

Reply:
I also gave some more descriptive titles to the info boxes

![image](https://github.com/CSCfi/csc-user-guide/assets/91546703/e7395a0d-1359-44ab-831f-05e2727a5e37)

Reply:
I think this is a good point by @joonas-somero 
Admonitions should be used to draw special attention, and if they are used too much, it seems to defeat the purpose (user is expected to pay special attention to "everything"). In this sense, the horizontal rule would be better. Perhaps custom CSS is overkill?